package.json

```diff
@@ -53,5 +53,8 @@
     "format": "tsfmt -r"
   },
   "author": "codice",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "chai-match": "^1.1.1"
```

Comment:
❓ Should this be under `"devDependencies"` instead? Do we need this at runtime or only to execute tests?

Reply:
Only tests